.travis.yml

```diff
@@ -1,19 +1,49 @@
----
 language: node_js
+dist: trusty
 node_js:
 - '6.11.4'
 
 branches:
   only:
     - master
     - /^greenkeeper/.*$/
+    - /^v?[0-9]+.[0-9]+.[0-9]+(-.*)?$/
 
-env:
-  - MONGODB_VERSION=3.2.13
+cache:
+  directories:
+  - node_modules
+  - "$HOME/.mongodb/versions"
 
-services: mongodb
-before_script:
-- npm install -g mongodb-runner
-- mongodb-runner start
-script: npm test -- --maxWorkers=4 && ./run_integration.sh
-after_script: cat ./coverage/coverage-final.json | ./node_modules/codecov.io/bin/codecov.io.js && rm -rf ./coverage
+jobs:
+  include:
+    - stage: test
+      env:
+      - MONGODB_VERSION=3.2.13
+      before_script:
+      - npm install -g mongodb-runner
+      - mongodb-runner start
+      script: npm test -- --maxWorkers=4 && ./run_integration.sh
+      after_script: cat ./coverage/coverage-final.json | ./node_modules/codecov.io/bin/codecov.io.js && rm -rf ./coverage
+    # release on github latest branch
+    - stage: release
+      env: 
+      before_script: skip
+      after_script: skip
+      script: npm run release_docs
+      deploy:
+        - provider: pages
+          skip_cleanup: true
+          github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
+          local_dir: docs/
+          on:
+            all_branches: true
+        - provider: npm
```

Comment:
Me Gusta! Awesome that this would cut out a step we would have to do by hand 👍 .

Reply:
Yes! Figured we should do as much as we could!